### PR TITLE
Add Jitpack badges to README

### PR DESCRIPTION
**Pull request by yasanglass**
This PR adds a few simple Jitpack badges to the README file:

- Build workflow status
- Latest Jitpack version (with status)
- Downloads per month

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
 # Opto
 
+[![Build](https://github.com/patrykandpatrick/opto/workflows/Build/badge.svg)](https://github.com/patrykandpatrick/opto/actions/workflows/build.yml)
+[![](https://jitpack.io/v/patrykandpatrick/opto.svg)](https://jitpack.io/#patrykandpatrick/opto)
+[![](https://jitpack.io/v/patrykandpatrick/opto/month.svg)](https://jitpack.io/#patrykandpatrick/opto)
```
Comment (patrickmichalik) [inline README.md:5]: I’ve navigated to this URL on a few devices, and the badge either takes a long time to load or doesn’t load at all. Thus, I’d suggest removing it.

```suggestion
```

File: README.md [hunk 2]
```diff
@@ -1,4 +1,8 @@
 # Opto
 
+[![Build](https://github.com/patrykandpatrick/opto/workflows/Build/badge.svg)](https://github.com/patrykandpatrick/opto/actions/workflows/build.yml)
```
Comment (patrickmichalik) [inline README.md:3]: I’d say “Build” is quite general and might be slightly confusing. How about this?

```suggestion
[![Build status](https://github.com/patrykandpatrick/opto/workflows/Build/badge.svg)](https://github.com/patrykandpatrick/opto/actions/workflows/build.yml)
```

File: README.md [hunk 3]
```diff
@@ -1,4 +1,8 @@
...
+[![](https://jitpack.io/v/patrykandpatrick/opto.svg)](https://jitpack.io/#patrykandpatrick/opto)
```
Comment (patrickmichalik) [inline README.md:4]: I’d suggest adding alternative text here too.

```suggestion
[![Latest version](https://jitpack.io/v/patrykandpatrick/opto.svg)](https://jitpack.io/#patrykandpatrick/opto)
```

**Review by patrickmichalik — CHANGES_REQUESTED** [review]
Hello, and thanks for the PR!

File: README.md (hunk 1 above)
Comment (yasanglass) [reply README.md]: I noticed that but I do not have the same issue on my projects that have it added on the README. I suspect it gets cached if its used more frequently or something like that. Removed it anyway.

Comment (patrickmichalik) [reply README.md]: Ah, got it. We’ll keep that in mind and potentially re-add the badge. Thanks!

**Review by patrickmichalik — APPROVED** [review]
Cheers!

**Pull request merged**